php/libraries/NDB_Client.class.inc

```diff
@@ -119,7 +119,9 @@ class NDB_Client
             . $config_additions
         );
         // start php session
-        session_start();
+        if (session_status() == PHP_SESSION_NONE) {
+            session_start();
+        }
```

Comment:
@AnyhowStep How is this related to the pull request?

Reply:
@driusian someone should issue this seperately into the major branch and this should go to minor